examples/dropwizard/pom.xml

```diff
@@ -0,0 +1,235 @@
+            <plugin>
+                <groupId>com.coderplus.maven.plugins</groupId>
+                <artifactId>copy-rename-maven-plugin</artifactId>
+                <version>1.0</version>
+                <executions>
+                    <execution>
+                        <id>copy-file</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <sourceFile>${dropwizard.server.config}</sourceFile>
+                            <destinationFile>${jib.extraContext}/${dropwizard.server.config}</destinationFile>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <!-- NOTE: Workaround for cleaning populated src/main/jib contents since jib.extraDirectory is incubating -->
+            <plugin>
+                <artifactId>maven-clean-plugin</artifactId>
+                <version>3.1.0</version>
+                <configuration>
+                    <filesets>
+                        <fileset>
+                            <directory>src/main/jib</directory>
+                            <includes>
+                                <include>${dropwizard.server.config}</include>
+                            </includes>
+                        </fileset>
+                    </filesets>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>${jib-maven-plugin.version}</version>
+                <configuration>
+                    <to>
+                        <image>${jib.to.image}</image>
+                        <tags>
+                            <tag>${project.version}</tag>
+                            <tag>latest</tag>
+                        </tags>
+                    </to>
+                    <container>
+                        <args>
+                            <arg>server</arg>
+                            <arg>${dropwizard.server.config}</arg>
+                        </args>
+                        <ports>
+                            <port>8080</port>
+                        </ports>
+
+                    </container>
+                    <extraDirectory>
+                        <!-- NOTE: Incubating feature. Defaulting to src/main/jib -->
+                        <path>${jib.extraContext}</path>
+                    </extraDirectory>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>dockerBuild</id>
+                        <goals>
+                            <goal>dockerBuild</goal>
+                        </goals>
```

Comment:
Looks like this is attaching `dockerBuild` to `package` and `build` to `deploy` - is it the intention to build to both the Docker daemon and the docker registry upon each `mvn deploy`?